systemwidealerts.md

```diff
@@ -0,0 +1,80 @@
+# System-wide alerts
+[Back to the list of all defined endpoints](endpoints.md)
+
+This endpoint exposes system-wide alerts to be shown as banners in the frontend.
+Only administrators are allowed to create and edit these alerts, while all other users can view them.
+
+This feature will be expanded in the future
+* DSpace currently only supports a single system-wide alert, but the endpoints are designed to accomodate multiple alerts
+* Alerts can specify whether/how user sessions should be restricted while the alert is active, but this is a placeholder as session control has not been implemented yet.
+
+
+## Retrieve all alerts
+**GET /api/system/systemwidealerts**
+This endpoint returns a list of all system-wide alerts.
```

Comment:
This is restricted to Admin-Only, so we should add a note about that to the contract.  Only Admins can see all alerts.

Reply:
Done.